package.json

```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "posix-getopt": "~1.0.0",
-    "hue.js": "https://github.com/bahamas10/hue.js/tarball/dave-1432672068",
+    "hue.js": "https://github.com/mattbucci/hue.js/tarball/master",
```

Comment:
my changes have been merged to "hue.js": "https://github.com/bahamas10/hue.js/tarball/master", as well now. 

Looks like we have some redundant work but it seems 👍 that they're roughly the same